src/kili/entrypoints/mutations/project/helpers.py

```diff
@@ -8,9 +8,11 @@ def verify_argument_ranges(
     consensus_tot_coverage: Optional[int],
     min_consensus_size: Optional[int],
     review_coverage: Optional[int],
-):
-    """Ensure that all arguments are set in a correct range or raise
-    a ValueError otherwise."""
+) -> None:
+    """Ensure that all arguments are set in a correct range.
+
+    Raise a ValueError otherwise.
+    """
```

Comment:
just to inform because it is not a big deal: the google style doctrings has a field dedicated to error raised: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
```
"""
Func Description

Args:

Returns:

Raises:
"""
```